task/sql-tasks.js

```diff
@@ -235,7 +366,15 @@ async function task_1_17(db) {
  *
  */
 async function task_1_18(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT	T.OrderDate, COUNT(*) AS 'Total Number of Orders'
+        FROM
+            (SELECT	DATE_FORMAT(OrderDate, '%Y-%m-%d %T') AS 'OrderDate'

```

Comment:
You can make it with one `group by` and `HAVING` operator. Without inner SELECT request